src/pages/install/index.md

```diff
@@ -1,134 +1,19 @@
-     <Tabs>
-     <TabItem value="gui" label="Graphically">
-
-       1. Open Control Panel
-       2. Click "User Accounts"
-       3. Click "User Accounts" again
-       4. Click "Change my environment variables"
-       5. Select the "Path" line in the **top** panel
-       6. Click "Edit..."
-       7. Click "Browse...", select the folder the files are in, and click OK
-       8. Make sure that the new entry (which should be highlighted) is at the bottom of the list; if not, click on "Move Down" until it is
-       9. Click "OK"
-       10. Click "OK"
-
-     </TabItem>
-     <TabItem value="cmd.exe" label="cmd.exe">
-
-       Run the following, replacing `<rgbds_path>` with the path to the directory that contains `rgbasm.exe`, `rgblink.exe`, etc.
-
-       ```cmd
-       setx PATH "%PATH%<rgbds_path>;"
-       ```
-
-       ...then open a new window for the changes to take effect.
-
-       If you only want to modify the PATH temporarily, instead of the permanent [`setx` command](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), you can use  the **temporary** [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1).
-
-     </TabItem>
-     <TabItem value="pwsh" label="PowerShell">
-
-       Run the following, replacing `<rgbds_path>` with the path to the directory that contains `rgbasm.exe`, `rgblink.exe`, etc.
-
-       ```cmd
-       setx PATH ${Env:PATH}<rgbds_path>;
-       ```
-
-       ...then open a new window for the changes to take effect.
-
-       If you only want to modify the PATH temporarily, instead of the permanent [`setx` command](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), you can use  the **temporary** [`set`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1).
-
-     </TabItem>
-     </Tabs>
-
-   - ...or put all of the files in your project's directory.
-   - ...or put all of the files in a directory already in the `PATH`.
-5. Profit! RGBDS can now be used from your favorite command line (`cmd.exe` or PowerShell, most likely). You can test it by running `rgbasm --version`.
+Most people will want to use a stable release (the more recent the better).
 
-</TabItem>
-</Tabs>
+These are the operating systems for which pre-built executables are available:
+- [Linux](/install/linux)
+- [macOS](/install/macOS)
+- [Windows](/install/windows)
 
-## Other
+If none of these options fits your needs, you can [build from source](/install/source).
 
 ### Docker
 
-We distribute an [official container image for RGBDS](https://github.com/gbdev/rgbds/pkgs/container/rgbds). It contains the built executables and the build dependencies in case you want to compile from source.
+We distribute an [official container image for RGBDS](https://github.com/gbdev/rgbds/pkgs/container/rgbds).
+It contains the built executables **and** the build dependencies, in case you want to compile from source.
```

Comment:
Italics for emphasis, bold for keywords or sometimes strong emphasis?
```suggestion
It contains the built executables *and* the build dependencies, in case you want to compile from source.
```